Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:10
+# docker build -t sourcecred .
+# available at vanessa/sourcecred if you don't want to build
+RUN apt-get update && \
+    apt-get install -y apt-transport-https python
+RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && \
```

Comment:
I’m no Docker expert, but is `curl | apt-key add -` really the safest
way to do this? Adding data from the internet as _package signing keys_
seems pretty alarming.

Reply:
Installing yarn with the package manager (as I was originally doing) we have to add the key to get the package. But since I'm using the install script later, there is a good chance I can just remove this now. Testing now!

Reply:
Good catch here! We're good without the extra step.